src/classes/CW_AudienceGroupTriggerHandler.cls

```diff
@@ -0,0 +1,219 @@
+/**
+* File Name :   CW_AudienceGroupTriggerHandler.cls
+* Description : D-106803
+*
+* Modification Log
+* ============================================================================
+* Ver Date	   Author		 Modification
+* --- ---------- -------------- --------------------------
+* 0.1 24/04/2014 AleX Panka      Initial version
+* 0.2 25/04/2014 Klea Kolaric    B-111670
+* 0.3 03/06/2014 Klea Kolaric    B-114503
+*
+*
+*/
+//Cristina P: 25/06/14: Changing the class from "with sharing" to "without sharing" 
+//so the user can create related records while cost/brief is in approval
+public without sharing class CW_AudienceGroupTriggerHandler {
+	
+	public static final String SEPARATOR = '_';
+	
+	public static void OnBeforeUpdateValidation(List<CW_Audience_Group__c> oldRecs, Map<Id,CW_Audience_Group__c> newRecs){
```

Comment:
Methods should use the `lowerCamelCase` naming convention. I suggest rename accordingly.